README.md

```diff
@@ -87,14 +87,22 @@ The method for installing *Turi Create* follows the
 To create a Python virtual environment called `venv` follow these steps:
 
 ```shell
-# Create a Python virtual environment
+# Create a Python virtual environment on Windows
```

Comment:
I don't think this style of file path will work in WSL (which is our currently supported Windows environment). WSL is a Linux subsystem on top of Windows, and uses Linux-style file paths including `~` for home directory, so the same instructions as for Linux and macOS should work. Have you tried the existing instructions there?